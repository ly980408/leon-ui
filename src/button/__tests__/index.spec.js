import BasicUse from '../example/BasicUse.vue'
import { mount } from '@vue/test-utils'

test('test button basic use', () => {
  const wrapper = mount(BasicUse)
  expect(wrapper.html()).toMatchSnapshot()
  wrapper.unmount()
})
