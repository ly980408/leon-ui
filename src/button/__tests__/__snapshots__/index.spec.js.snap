// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test button example 1`] = `"<button class=\\"l-button l-button--default l-button--normal\\"><span class=\\"l-button__content\\">默认按钮</span></button><button class=\\"l-button l-button--primary l-button--normal\\"><span class=\\"l-button__content\\">主要按钮</span></button><button class=\\"l-button l-button--text l-button--normal\\"><span class=\\"l-button__content\\">文字按钮</span></button><button class=\\"l-button l-button--default l-button--normal is-round\\"><span class=\\"l-button__content\\">圆角按钮</span></button>"`;
